src/Uno.UI/UI/Xaml/Controls/ListViewBase/VirtualizingPanelLayout.managed.cs

```diff
@@ -1191,29 +1175,19 @@ private void ResetReorderingIndex()
 		/// </summary>
 		private protected class Line
 		{
-			public (FrameworkElement container, Uno.UI.IndexPath index)[] Items { get; }
-			public Uno.UI.IndexPath FirstItem { get; }
-			public Uno.UI.IndexPath LastItem { get; }
+			public Uno.UI.IndexPath Item { get; }
 			public int FirstItemFlat { get; }
```

Comment:
I'm not sure to understand why you removed the list of items. What's the intent here?

Reply:
@jeromelaban Because it's always a single element array. This is the only constructor call:

https://github.com/unoplatform/uno/blob/2af7b1a6953cc501dd6b16c07c4510fd8a937828/src/Uno.UI/UI/Xaml/Controls/ItemsStackPanel/ItemsStackPanelLayout.managed.cs#L34

It only provides a single item

Reply:
I understand. The original intent was to support grouping which is why the implementation is done this way. I think we should keep this regardless.

Reply:
In that case, yes sure, as long as it will help in future.